superset/commands/dashboard/importers/v1/utils.py

```diff
@@ -142,6 +142,12 @@ def update_id_refs(  # pylint: disable=too-many-locals
                 id_map[old_id] for old_id in scope_excluded if old_id in id_map
             ]
 
+        charts_in_scope = native_filter.get("chartsInScope", [])
+        if charts_in_scope:
```

Comment:
```suggestion
        if charts_in_scope := native_filter.get("chartsInScope", []):
```

Reply:
@rdubois-kh curious your take on the suggestion, and @john-bodley wondering if you see this as blocking.

Reply:
@rusackas thanks for the heads-up. I took the suggestion into account in this https://github.com/apache/superset/pull/26389/commits/1fab8976ca643c8e5a5b9ae1a8f8e18ff7c8b639